wondrous-app/types/task.ts

```diff
@@ -85,4 +88,5 @@ export interface TaskInterface {
   parentTaskId?: string;
   totalSubtaskCount?: number;
   isProposal?: boolean;
+  paymentStatus: ValueOf<typeof PAYMENT_STATUS> | null;
```

Comment:
nice

Reply:
I think it's better to make it enum, but need to check

Reply:
We definitely can't change it for enum atm, because it will cause waterfall refactoring, but i think it's good for now